api/controllers/media.js

```diff
@@ -11,16 +9,33 @@ async function uploadMedia(req, res) {
 
   try {
     const uploadedFile = req.files.file[0];
+
+    /* 
+    NOTICE !! 
+    you have to have the 
+    "CBOARD_CONTAINER_NAME=????" variable in your environment variables
+    */
+    var containerName = process.env.CBOARD_CONTAINER_NAME;
```

Comment:
I would still make it `const` and upper case the variable name as a constant

Reply:
done